packages/forms/src/UIForm/UIForm.component.js

```diff
@@ -215,7 +215,20 @@ export class UIFormComponent extends React.Component {
 		const { mergedSchema } = this.state;
 		const { properties, customValidation } = this.props;
 		const newErrors = validateAll(mergedSchema, properties, customValidation);
-		const errors = { ...this.props.errors, ...newErrors };
+		Object.keys(this.props.errors).forEach(errorKey => {
```

Comment:
If i understand, you want keet the reference, can you do that ?
```suggestion

		const errors = Object.assign(this.props.errors, newErrors);
```

Reply:
no because errors is define with only keys present on newErrors to skip potential new hidden field with error